src/atoms.js

```diff
@@ -49,7 +50,7 @@ export const addPrompt = atom(null, (get, set, prompt) => {
     set(chatHistoryAtom, (prev => [...prev, makeInputMessage(query)]));
   }
 
-  let queryUrl = appType === "alerting" ? "https://api.zeno.ds.io/stream/dist_alert" : "https://api.zeno.ds.io/stream/kba";
+  let queryUrl = appType === "alerting" ? "https://dev.api.zeno.ds.io/stream/dist_alert" : "https://dev.api.zeno.ds.io/stream/kba";
```

Comment:
Id' suggest to make this an object:

```
{
  alerting: "https://dev.api.zeno.ds.io/stream/dist_alert",
  monitoring: ""https://dev.api.zeno.ds.io/stream/dist_alert""
}
```

Just in case you want to add more app types later.